pillar/edx/ansible_vars/next_residential.sls

```diff
@@ -9,3 +9,22 @@ edx:
     EDXAPP_SESSION_COOKIE_NAME: {{ environment }}-{{ purpose }}-session
     EDXAPP_CMS_AUTH_EXTRA:
       SECRET_KEY: __vault__:gen_if_missing:secret-residential/global/edxapp-lms-django-secret-key>data>value
+    EDXAPP_REGISTRATION_EXTRA_FIELDS:
+      confirm_email: "hidden"
+      level_of_education: "optional"
+      gender: "optional"
+      year_of_birth: "optional"
+      mailing_address: "hidden"
+      goals: "optional"
+      honor_code: "required"
```

Comment:
Is this required or isn't it assumed that if they're students here that they already abide by an `honor_code`?

Reply:
I was just copying from what we have in xpro so that I can eliminate the need to enter the country for adding users.